app/controllers/application_controller.rb

```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
+
+   before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
Excuse me, what is this monstrous extra space?
```suggestion
  before_action :configure_permitted_parameters, if: :devise_controller?
```